packages/reports/addon/components/report-actions/export.ts

```diff
@@ -43,7 +44,8 @@ export default class ReportActionExport extends Component<Args> {
    * filename for the downloaded file
    */
   get filename() {
-    return this.args.model.title;
+    const dateString = moment().format('YYYYMMDDTHHmmss');
+    return `${this.args.model.title}-${dateString}.csv`;
```

Comment:
since `filename` might be used for other formats as well, can we remove the `.csv` here and just add it in `downloadTask`?